source/versioned-api/tests/crud-api-version-1.yml

```diff
@@ -209,7 +209,9 @@ tests:
                 key: x
                 <<: *expectedApiVersion
 
-  - description: "estimatedDocumentCount appends declared API version"
+  - description: "estimatedDocumentCount appends declared API version on less than 4.9.0"
+    runOnRequirements:
+      - maxServerVersion: "4.9.0"
```

Comment:
This is also inclusive and should probably be "4.8.99". See: https://github.com/mongodb/specifications/blob/master/source/unified-test-format/unified-test-format.rst#runonrequirement